app/src/main/java/de/danoeh/antennapod/fragment/CoverFragment.java

```diff
@@ -217,6 +341,26 @@ private void configureForOrientation(Configuration newConfig) {
                 textParams.weight = 1;
                 imgvCover.setLayoutParams(params);
             }
+
+            spacerVisible = false;
+            detailsParent = textContainer;
+            detailsWidth = ViewGroup.LayoutParams.WRAP_CONTENT;
+        }
+
+        if (displayedChapterIndex == -1) {
+            detailsWidth = ViewGroup.LayoutParams.WRAP_CONTENT;
+        }
+
+        spacer.setVisibility(spacerVisible ? View.VISIBLE : View.GONE);
+        counterweight.setVisibility(spacerVisible ? View.VISIBLE : View.GONE);
+        LinearLayout.LayoutParams wrapHeight =
+                new LinearLayout.LayoutParams(detailsWidth, ViewGroup.LayoutParams.WRAP_CONTENT);
+        episodeDetails.setLayoutParams(wrapHeight);
+        getView().findViewById(R.id.vertical_divider).setVisibility(spacerVisible ? View.GONE : View.VISIBLE);
+
+        if (episodeDetails.getParent() != detailsParent) {
+            ((ViewGroup) episodeDetails.getParent()).removeView(episodeDetails);
+            detailsParent.addView(episodeDetails);
```

Comment:
Moving around a View seems a bit strange to me. Why can't it just always stay in the textContainer? It is always displayed below the text, anyway.

Reply:
the spacing doesnt work in portait* if its in the textcontainer, i dont think its an issue, why would it be? Works well and requieres the least amount of code :)

Reply:
Okay, let's keep it like this for now.